From 44fd486de59fbfb0416dd69d6d9bc7715c23e9c5 Mon Sep 17 00:00:00 2001
From: Gustavo Ornaghi Antunes <gustavo.ornaghiantunes@windriver.com>
Date: Mon, 12 Jun 2023 14:14:23 -0300
Subject: [PATCH] csm-observability: remove unnecessary images

This patch consists of commenting the csm metrics images that
is not supported by the app yet.
This prevent the images are automatically downloaded
by the system.

Signed-off-by: Gustavo Ornaghi Antunes <gustavo.ornaghiantunes@windriver.com>
Signed-off-by: Hediberto Cavalcante da Silva <hediberto.cavalcantedasilva@windriver.com>
---
 charts/karavi-observability/values.yaml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/charts/karavi-observability/values.yaml b/charts/karavi-observability/values.yaml
index 135c1db..4c9ebce 100644
--- a/charts/karavi-observability/values.yaml
+++ b/charts/karavi-observability/values.yaml
@@ -16,7 +16,7 @@ karaviTopology:
     probability: 0.0
 
 karaviMetricsPowerflex:
-  image: dellemc/csm-metrics-powerflex:v1.5.0
+  #image: dellemc/csm-metrics-powerflex:v1.5.0
   enabled: true
   collectorAddr: otel-collector:55680
   # comma separated list of provisioner names (ex: csi-vxflexos.dellemc.com)
@@ -82,7 +82,7 @@ karaviMetricsPowerstore:
     probability: 0.0
 
 karaviMetricsPowerscale:
-  image: dellemc/csm-metrics-powerscale:v1.2.0
+  #image: dellemc/csm-metrics-powerscale:v1.2.0
   enabled: true
   collectorAddr: otel-collector:55680
   # comma separated list of provisioner names (ex: csi-isilon.dellemc.com)
@@ -132,7 +132,7 @@ karaviMetricsPowerscale:
     skipCertificateValidation: true
 
 karaviMetricsPowermax:
-  image: dellemc/csm-metrics-powermax:v1.0.0
+  #image: dellemc/csm-metrics-powermax:v1.0.0
   enabled: true
   collectorAddr: otel-collector:55680
   # comma separated list of provisioner names (ex: csi-powermax.dellemc.com)
-- 
2.25.1
